NCS.DSS.Sessions/NCS.DSS.Sessions.csproj

```diff
@@ -1,37 +1,44 @@
 ﻿<Project Sdk="Microsoft.NET.Sdk">
   <PropertyGroup>
-    <TargetFramework>net6.0</TargetFramework>
+    <TargetFramework>net8.0</TargetFramework>
     <AzureFunctionsVersion>v4</AzureFunctionsVersion>
+    <OutputType>Exe</OutputType>
+    <ImplicitUsings>enabled</ImplicitUsings>
```

Comment:
As per Bens previous comments on other repos, `enabled` needs to be changed to `enable`